onnxruntime/core/optimizer/utils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/graph/onnx_protobuf.h"
+
+namespace onnxruntime {
+namespace optimizer {
```

Comment:
Wondering if we want to unify this namespace with the transformer_utils. They are both utils methods for the optimizer. And maybe optimizer_utils (or graph_optimizer_utils) is a better name that transformer_utils.

Reply:
Updated to optimizer_utils